backend/generator/user/tests/test_creating_users.py

```diff
@@ -0,0 +1,47 @@
+import pytest
+from django.contrib.auth import get_user_model
+
+
+@pytest.mark.django_db
+def test_create_user():
+    User = get_user_model()
+    user = User.objects.create_user(email='normal@user.com', password='foo')
+    assert user.email == 'normal@user.com'
+    assert user.is_active
+    assert not user.is_staff
+    assert not user.is_superuser
+
+    try:
+        # username is None for the AbstractUser option
+        # username does not exist for the AbstractBaseUser option
+        assert user.username is None
+    except AttributeError:
+        pass

```

Comment:
получится здесь сделать вот так?
```suggestion
   with pytest.raises(AttributeError):
        user.username
```

Reply:
+ это можно выделить в самостоятельные тест, который будет тестировать именно отсутствие юзернейма

Reply:
Не получится сделать, как я предложил?

Reply:
не, там ошибки